### ISSUE TITLE
What is this pin for?

### ISSUE THREAD

**Issue by SarcasmRules** [nevermore3d/StealthMax#8]
What is this pin for in the Flow_Chamber.stl?
<img width="279" alt="Screenshot 2023-08-09 at 08 14 54" src="https://github.com/nevermore3d/StealthMax/assets/31560665/01cd7411-b19d-4637-a64a-fc7b72530bc2">

**Comment by 0ndsk4** [comment]
It’s a spacer for when stacking the VOC sensors. Makes it a bit more stable.

**Comment by SarcasmRules** [comment]
Thx, much appreciated. 
One more question, there are 2 of each sensors on the BOM but only one of each is in the CAD, assuming both are intake. In the controller section by SanaaHamel there is an intake and exhaust sensor(s); where are the other two installed?

**Comment by 0ndsk4** [comment]
There are two sensor pairs. A BME280 and a SGP40 are stacked on top of each other - one pair at the intake (flow chamber), one pair at the exhaust (center hub next to the pico).

**Comment by SarcasmRules** [comment]
That's what I was missing, thanks! :) I'm printing one up, have the flow and main chamber done, printing the basket now.
It looks nice but way bigger than I thought (TWSS!).

**Comment by mrzottel** [comment]
So just to be sure - those two holes left and right of the pico are used for mounting the exhaust sensors?

<img width="810" alt="image" src="https://github.com/nevermore3d/StealthMax/assets/6313125/1211237c-0452-46ed-a4ba-db13da9f9ee8">

**Comment by 0ndsk4** [comment]
Yes. Both are not needed tho, most would do a dual sensor stack using a single mounting hole. Saves on wiring.
